### ISSUE TITLE
[KEYBOARD]: Unable to access Skip to Content link in NVDA

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#3843]
## Description

**[KEYBOARD]: Unable to access Skip to Content link in NVDA**

Using Firefox and NVDA, am unable to access the Skip to Content link.

## Point of Contact

Jennifer

## Acceptance Criteria

As a keyboard user, I want to access the Skip to Content link so I can bypass the heading and navigation links, which are repeated throughout the site, so I may get to the primary content area of the page.

## Environment

* Operating System: Windows 10
* Browser: Firefox "latest"
* Screenreading device: NVDA
* Server destination: https://staging.va.gov/disability/view-disability-rating/rating/

## Steps to Recreate

1. Enter `https://staging.va.gov/disability/view-disability-rating/rating/` in Firefox browser
2. Start NVDA screenreading device
3. Navigate to the Skip to Content link
4. Verify the Skip to Content link is not available

## WCAG or Vendor Guidance (optional)

[WCAG 2.4.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html)

**Comment by jason-gcio** [comment]
While this has not been resolved it has been moved to the correct team to address it further.
#3673

**Comment by jenstrickland** [comment]
VA gov performance team is evaluating web performance impact on this functionality.

**Comment by 1Copenut** [comment]
@jenstrickland Are you still seeing this incorrect behavior, or can this be closed?

**Comment by jenstrickland** [comment]
@1Copenut — This issue is still happening.

**Comment by laflannery** [comment]
I do not believe this is still an issue - the above URL does not appear to exist anymore.  Will close in 1 week unless I hear otherwise
